pyqtgraph/opengl/items/GLGraphItem.py

```diff
@@ -0,0 +1,94 @@
+from OpenGL.GL import *
+from ..GLGraphicsItem import GLGraphicsItem
+from .GLScatterPlotItem import GLScatterPlotItem
+from ...Qt import QtGui
+from ... import functions as fn
+
+
+__all__ = ['GLGraphItem']
+
+class GLGraphItem(GLGraphicsItem):
+    """A GLGraphItem displays graph information as
+    a set of nodes connected by lines (as in 'graph theory', not 'graphics').
+    Useful for drawing networks, trees, etc.
+    """
+
+    def __init__(self, **kwds):
+        GLGraphicsItem.__init__(self)
+
+        self.edges = None
+        self.edgeColor = (1.0, 1.0, 1.0, 0.5)
+        self.edgeWidth = 1.0
+
+        self.scatter = GLScatterPlotItem()
+        self.scatter.setParentItem(self)
+        self.setData(**kwds)
+
+
+    def setData(self, **kwds):
+        """
+        Change the data displayed by the graph. 
+        ==============  =======================================================================
+        **Arguments:**
+        edges           (M,2) array of connection data. Each row contains indexes
+                        of two nodes that are connected.
+        edgeColor       The color to use when drawing lines between connected
+                        nodes. AMay be one of:
+                        * QColor
+                        * an array [red, green blue, alpha]
+                        * None (to disable edge drawing)
+                        * [not available yet] a record array of length M
```

Comment:
I think this is a good feature to have; it's outside of my skill-set with openGL regarding how to implement.  I'll leave this comment as un-resolved, but leave the comment in there in case anyone wants to come along and suggest an implementation.